### PR TITLE
add webapp and server workflows for an auto-deployment

### PR DESCRIPTION
**Pull request by untari**
This pull request introduces two separate workflows for issues: (https://github.com/fossasia/eventyay-video/issues/94).

one for the webapp and another for the server component. The goal is to streamline the deployment process and ensure that updates to the webapp and server are independently and automatically deployed when changes are pushed to the repository.

### CODE REVIEW

File: .github/workflows/docker-server.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      
+      - name: Build and push Docker image
+        uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
+        with:
+          context: ./server
+          dockerfile: Dockerfile-dev
+          push: true
+          tags: eventyay/eventyay-video:development-webapp
```
Comment (marcoag) [inline .github/workflows/docker-server.yml:27]:
```suggestion
          tags: eventyay/eventyay-video:development-server
```

**Review by marcoag — CHANGES_REQUESTED** [review]

**Review by marcoag — APPROVED** [review]

**Comment by marcoag** [comment]
Ok, let's merge and run the workflows.

**Comment by marcoag** [comment]
I realized both workflows have the same name, @untari please fix on a new PR.

**Comment by marcoag** [comment]
Workflow fails, it seems to not be able to find the dockerfile. @untari ptal https://github.com/fossasia/eventyay-video/actions/runs/9258277324/job/25468072107

**Comment by untari** [comment]
> Workflow fails, it seems to not be able to find the dockerfile. @untari ptal https://github.com/fossasia/eventyay-video/actions/runs/9258277324/job/25468072107

that's unfortunate, looking into it now. Thanks @marcoag

**Pull request merged**